libqtile/backend/wayland/core.py

```diff
@@ -631,6 +631,38 @@ def loop(wid):
         lib.qw_server_loop_visible_views(self.qw, loop)
         return wids
 
+    @expose_command()
+    def get_scene_graph(self):
```

Comment:
My PR for stacking in X11 has a similar exposed method in the X11 `Core` called `show_stacking_order`. Given these are both doing effectively the same thing, can we find a name that works across both backends? `scene tree` is a bit wayland specific.